protoc-gen-grpc-gateway/BUILD.bazel

```diff
@@ -0,0 +1,40 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_binary", "go_library")
+load("@io_bazel_rules_go//proto:compiler.bzl", "go_proto_compiler")
+
+package(default_visibility = ["//visibility:private"])
+
+go_library(
+    name = "go_default_library",
+    srcs = ["main.go"],
+    importpath = "github.com/grpc-ecosystem/grpc-gateway/protoc-gen-grpc-gateway",
+    deps = [
+        "//protoc-gen-grpc-gateway/descriptor:go_default_library",
+        "//protoc-gen-grpc-gateway/gengateway:go_default_library",
+        "@com_github_golang_glog//:go_default_library",
+        "@com_github_golang_protobuf//proto:go_default_library",
+        "@com_github_golang_protobuf//protoc-gen-go/plugin:go_default_library",
+    ],
+)
+
+go_binary(
+    name = "protoc-gen-grpc-gateway",
+    embed = [":go_default_library"],
+)
+
+go_proto_compiler(
+    name = "go_gen_grpc_gateway",
+    options = ["logtostderr=true"],
```

Comment:
I'm not sure if we should use the option `logtostderr=true`. Bazel seems to succeed if the code gen fails.

Reply:
Do you have any theories on that? I would expect `protoc` to propagate the nonzero exit code through.

Reply:
It works properly. I was having another issue that made me think this was the cause. Not true.